README.md

```diff
@@ -5,7 +5,56 @@ entry point, also known as a [Relying Party](https://www.ietf.org/archive/id/dra
 in [RATS](https://datatracker.ietf.org/doc/draft-ietf-rats-architecture/)
 role terminology.
 
+KBS integrates the [Attestation-Service](https://github.com/confidential-containers/attestation-service) to verify TEE evidence.
+
+KBS can also be deployed as [Verifier](https://www.ietf.org/archive/id/draft-ietf-rats-architecture-22.html) in RATS role terminology.
+In this case, KBS will be responsible for distributing the Attestation Token (Passport).
+
 ## Protocol
 
 The KBS implements and supports a simple, vendor and hardware-agnostic
 [implementation protocol](https://github.com/confidential-containers/kbs/blob/main/docs/kbs_attestation_protocol.md).
+
+## API
+
+KBS API is implemented based on HTTP/HTTPS. OpenAPI 3.1 description document:
+- [kbs.yaml](docs/kbs.yaml)
+
+## Usage
+
+### Build and Run
+
+Start KBS and specify the address it listens to (take `127.0.0.1:8080` as an example):
+
+```shell
+make kbs
+./target/debug/kbs --socket 127.0.0.1:8080
+```
+
+Start with customized [config](src/config.rs):
+
+```shell
+./target/debug/kbs --socket 127.0.0.1:8080 --config /path/to/config.json
+```
+
+## Resource Repository
+
```

Comment:
Here I would add one paragraph explaining what a resource repository is.

Reply:
Good suggestion, I will add this.